src/main/webapp/resources/js/components/lists/BasicList.jsx

```diff
@@ -21,7 +21,8 @@ const { Text } = Typography;
 
 export function BasicList({
   itemLayout = "horizontal",
-  dataSource = { dataSource }
+  dataSource = { dataSource },
+  customClassNames = false
```

Comment:
I don't think you need this, plus it is really smelly.  If you want to what I would to is add a `props` attribute to the object and then spread it.  That way you will automatically get the class name you want:

Calling code: 
```javascript
{
      title: i18n("FastQC.maxLength"),
      desc: fastQC.maxLength,
      props: {
		className: "t-fastqc-max-length", 
		// whatever other props like style, ect...
	 }
},
```
And then in you BasicList:
```javascript
 <List.Item.Meta
            title={<Text strong>{item.title}</Text>}
            description={item.desc}
            {...item.props}
          />
```

Reply:
Updated in 8d915d2b5135ef05b2f4eafaebe586d755b13903